### DISCUSSION TITLE
how to solve?

### DISCUSSION THREAD

**Discussion by alphagui** [l0o0/jasminum#322] [Q&A]
1729583777542	addons.webextension.jasminum@linxzh.com	WARN	Loading extension 'jasminum@linxzh.com': Reading manifest: Warning processing version: version must be a version string consisting of at most 4 integers of at most 9 digits without leading zeros, and separated with dots

**Comment by northword** [comment]
无影响，忽略即可。

**Reply by alphagui** [reply]
windows版的没问题，macbook就用不了，是报这个错？不知道大家有没有遇到这个问题

**Reply by northword** [reply]
与此无关。

用不了是什么用不了，现象是什么，如何复现。
